resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -1101,6 +1101,7 @@ sqs.delete.message.failed=Failed to delete {0, choice, 1#message|2#{0,number} me
 sqs.delete.message.partial_successs=Failed to delete some messages! {0, choice, 1#message|2#{0,number} messages} were deleted.
 sqs.delete.message.succeeded=Successfully deleted {0, choice, 1#1 message|2#{0,number} messages}.
 sqs.delete.queue.action=Delete Queue...
+sqs.delete.warning_text=Deletion of the queue may take more than a minute to be reflected
```

Comment:
how about we make this more generic? i'm sure there's other resource deletions that we have where delete may take a while

Reply:
Changed the message to be more generic and accept the resourceType as parameter